### ISSUE TITLE
MultiSelect.Filterable gets id reference violation in DAP

### ISSUE THREAD

**Issue by marcyb1966** [carbon-design-system/carbon#3941]
## Environment

> Operating system: Mac

> Browser: Chrome

> Automated testing tool and ruleset: DAP

> Assistive technology used to verify

## Detailed description

> What version of the Carbon Design System are you using? 10.4.1

> What did you expect to happen? 
Would receive no DAP violations when run against Multiselect.Filterable

> What happened instead?
Received the violation: 
![image](https://user-images.githubusercontent.com/51334642/64559073-0e540880-d30b-11e9-8a34-d64a1e42a3ce.png)

> What WCAG 2.1 checkpoint does the issue violate?
DAP violation - Id references used in WAI-ARIA properties must be valid
## Steps to reproduce the issue

1. Render MultiSelect.Filterable component with an id specified.
2. Run DAP
3. You will get violation mentioned above.

Sandbox example:
https://codesandbox.io/s/codesandbox-ey3to

> Please create a reduced test case in CodeSandbox
>
> - Style and vanilla JS:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/components/examples/codesandbox
> - React:
>   https://codesandbox.io/s/github/carbon-design-system/carbon/tree/master/packages/react/examples/codesandbox

## Additional information

- Screenshots or code
- Notes

**Comment by aledavila** [comment]
Closing false positive. It gives the violation because the menu isn't open and therefore can't find the matching id but it does exist.
